### PR TITLE
hw/marchenko07

### PR DESCRIPTION
**Pull request by Hort1934**

### CODE REVIEW

File: lesson07/hw/Marchenko/task_1.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        self.age = age
+    def compare_age(self):
+        #print(self.age)
+        if self.age > Me.age:
+            result = "older than"
+        elif self.age < Me.age:
+            result = "younger than"
+        elif self.age == Me.age:
+            result = "the same age as"
+        return (f"{self.name} is {result} me")
```
Comment (lhalam) [inline lesson07/hw/Marchenko/task_1.py]: метод має приймати іншу персону `def compare_age(self, person):`
print(p3.compare_age(Me))

**Review by lhalam — COMMENTED** [review]
виправити зауваження

File: lesson07/hw/Marchenko/task_1.py (hunk 1 above)
Comment (Hort1934) [reply lesson07/hw/Marchenko/task_1.py]: class Person:
    my_age = 25


    def __init__(self, name, age):
        self.name = name
        self.age = age


    def compare_age(self, other_person):
        if self.age > other_person.age:
            print(f"{self.name} is older than as me.")
        elif self.age < other_person.age:
            print(f"{self.name} is younger than me.")
        else:
            print(f"{self.name} is the same age as me.")


p1 = Person("Andriy", 18)
p2 = Person("Andriy1", 20)
p3 = Person("Andriy2", 19)
p2.compare_age(p1)
p2.compare_age(p3)
p1.compare_age(p1)

**Comment by Hort1934** [comment]
class Person:
my_age = 25

def __init__(self, name, age):
    self.name = name
    self.age = age


def compare_age(self, other_person):
    if self.age > other_person.age:
        print(f"{self.name} is older than as me.")
    elif self.age < other_person.age:
        print(f"{self.name} is younger than me.")
    else:
        print(f"{self.name} is the same age as me.")
p1 = Person("Andriy", 18)
p2 = Person("Andriy1", 20)
p3 = Person("Andriy2", 19)
p2.compare_age(p1)
p2.compare_age(p3)
p1.compare_age(p1)

**Comment by lhalam** [comment]
lesson06/hw/marchenko/task_1.py

**Comment by Hort1934** [comment]
Done.

**Pull request merged**